<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title_label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>600</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="text">
     <string>3D Point Cloud Object Detection</string>
    </property>
   </widget>
   <widget class="QPushButton" name="annotate_pushButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>600</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Annotation</string>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string>QPushButton { background-color: #007AFF; color: white; border-radius: 10px; }</string>
    </property>
   </widget>
   <widget class="QPushButton" name="aug_pushButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>600</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Augmentation</string>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string>QPushButton { background-color: #007AFF; color: white; border-radius: 10px; }</string>
    </property>
   </widget>
   <widget class="QPushButton" name="training_pushButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>290</y>
      <width>600</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Training</string>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string>QPushButton { background-color: #007AFF; color: white; border-radius: 10px; }</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inf_pushButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>360</y>
      <width>600</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Inference</string>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string>QPushButton { background-color: #007AFF; color: white; border-radius: 10px; }</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
